utils/hct/hctdb.py

```diff
@@ -5,6 +5,21 @@
 ###############################################################################
 import os
 
+def parse_command_line_options():
+    import argparse
+    parser = argparse.ArgumentParser(description="A script for outputting and searching through all Dxil Instructions")
+    
+    parser.add_argument(
+        '--query',
+        help='If given, stores a path to a query file. The option requires that the user create the specified file \
```

Comment:
I don't think this is the right approach.  Writing a script that imports hctdb.py and groks the instructions would be better.  If shared functionality is desired, add that to hctdb_instrhelp.py and import that from your query script instead.